CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -0,0 +1,110 @@
+  * Add [a per-rule example](https://github.com/antrea-io/antrea/blob/v1.6.0/docs/feature-gates.md#networkpolicystats) for NetworkpolicyStats docs . ([#3356](https://github.com/antrea-io/antrea/pull/3356), [@ceclinux])
+
+### Changed
+
+- [Windows] Use uplink MAC as source MAC when transmitting packets to underlay network from Windows Nodes. Security-related configurations like "Forged transmits" in VMware vSphere don't need to be degraded. ([#3516](https://github.com/antrea-io/antrea/pull/3516), [@wenyingd])
+- Use iptables-wrapper in Antrea container to support distros that runs iptables in "nft" mode. ([#3276](https://github.com/antrea-io/antrea/pull/3276), [@antoninbas])
+- Install cni conf after installing cni bins to support container runtime cri-o. ([#3154](https://github.com/antrea-io/antrea/pull/3154), [@tnqn])
+- Upgrade packaged Whereabouts version to v0.5.1. ([#3511](https://github.com/antrea-io/antrea/pull/3511), [@antoninbas])
+- Upgrade to go-ipfix v0.5.12. ([#3352](https://github.com/antrea-io/antrea/pull/3352), [@yanjunz97])
+- Upgrade Kustomize from v3.8.8 to v4.4.1 to fix Cronjob patching bugs. ([#3402](https://github.com/antrea-io/antrea/pull/3402), [@yanjunz97])
+- Fail in Agent initialization if GRE tunnel type is used with IPv6. ([#3156](https://github.com/antrea-io/antrea/pull/3156), [@antoninbas])
+- Refactor the OpenFlow pipeline for future extensibility. ([#3058](https://github.com/antrea-io/antrea/pull/3058), [@hongliangl])
+- Validate IP ranges of IPPool for Antrea IPAM. ([#2995](https://github.com/antrea-io/antrea/pull/2995), [@ksamoray])
+- Validate values of protocol and labelSelector in the crd schema of Antrea-native policies. ([#3342](https://github.com/antrea-io/antrea/pull/3342), [@KMAnju-2021])
+- Validate labels in ANP/ACNP/CG. ([#3331](https://github.com/antrea-io/antrea/pull/3331), [@GraysonWu])
+- Reduce permission of antrea service accounts. ([#3393](https://github.com/antrea-io/antrea/pull/3393), [@tnqn])
+- Remove Kind-specific manifest and scripts. Antrea now uses OVS kernel datapath for Kind clusters. ([#3413](https://github.com/antrea-io/antrea/pull/3413), [@antoninbas])
+- Remove --k8s-1.15 flag from hack/generate-manifest.sh. ([#3350](https://github.com/antrea-io/antrea/pull/3350), [@antoninbas])
+- Remove all legacy (*.antrea.tanzu.vmware.com) APIs. ([#3299](https://github.com/antrea-io/antrea/pull/3299), [@antoninbas])
+- Remove unnecessary CRDs and RBAC rules from Multi-cluster manifest. ([#3491](https://github.com/antrea-io/antrea/pull/3491), [@luolanzone])
+- Update label and image repo of antrea-mc-controller to be consistent with antrea-controller and antrea-agent. ([#3266](https://github.com/antrea-io/antrea/pull/3266) [#3466](https://github.com/antrea-io/antrea/pull/3466), [@luolanzone])
+- Add clusterID annotation to ServiceExport/Import resources. ([#3359](https://github.com/antrea-io/antrea/pull/3359), [@luolanzone])
```

Comment:
I feel this one is minor to include in the change log. No strong opinion.

Reply:
keep it as it's a change to API resources.

Reply:
Maybe next time we should group items of a feature if more than 3 items for the feature. It is not easy to understand individual small changes.